### PR TITLE
Overall review of SDK docs 3.0.2 - Closes #686

### LINKED ISSUE #686: Overall review of SDK docs 3.0.2
Overall review of entire documentation to include the structure, tone of voice, style, grammar, etc, etc..

Parent issue #665

### PR DESCRIPTION
**Pull request by chris529**
Closes #686

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please fix the conflicts first :)

**Comment by chris529** [comment]
Hi Mona, The conflicts seem to be resolved, although I seem to have a merging blocked message. Can you check this please when you have a minute? Many thanks.

File: modules/ROOT/pages/guides/app-development/custom-transactions.adoc [hunk 1]
```diff
@@ -262,7 +262,7 @@ class HelloTransaction extends BaseTransaction {
 == The Store
 
 The Store is responsible for the caching and accessing transaction and account data.
-The `store` is available inside the `prepare()`, `applyAsset()` and `undoAsset()` methods and provides methods to get and set the data from the database.
+The `store` is available inside the `prepare()`, `applyAsset()` and `undoAsset()` methods, and provides procedures to get and set the data from the database.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/custom-transactions.adoc]: methods is the correct terminology in this context

File: modules/ROOT/pages/guides/index.adoc [hunk 2]
```diff
@@ -89,14 +89,14 @@ All remaining topics of the Hello World application are explained further in the
...
+Each of these guides covers a specific topic of node management, together with providing examples and explaining the available options.
 
 The most important sections pertaining to the node management are described below:
 
 . The xref:{url_api_access}[API access guide] explains, how to manage the API access to a node.
-. The xref:{url_forging}[enable forging guide] explains how to enable, check, and disable forging on a node.
+. The xref:{url_forging}[enable forging guide] covers how to enable, check, and disable forging on a node.
 . In the xref:{url_enable_ssl}[enable SSL guide], the required essential parameters for enabling SSL security are covered, together with an example of SSL configuration.
-. The xref:{url_logging}[activate logging guide] describes the logging messages the log level destinations and rotations, for monitoring or debugging a node.
+. The xref:{url_logging}[activate logging guide] describes the logging messages, including the log level destinations and rotations, for monitoring or debugging a node.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]: remove including here
```suggestion
. The xref:{url_logging}[activate logging guide] describes the logging messages, the log level destinations and rotations, for monitoring or debugging a node.
```

File: modules/ROOT/pages/references/api-specification.adoc [hunk 3]
```diff
@@ -12,12 +12,12 @@ TIP: Check the xref:{url_core_api}[Lisk Core API] to make live API requests to t
 == Access restrictions
```
Comment (Tschakki) [inline modules/ROOT/pages/references/api-specification.adoc]: The contents here are autogenerated, and need to be updated in the `swagger.yml` file in the lisk-sdk repository, if required. For now, please revert all changes made in this file.

File: modules/ROOT/pages/references/changelog.adoc [hunk 4]
```diff
@@ -162,23 +162,23 @@ __optional__|Broadhash of the network|string (hex)|
 
 |{set:cellbgcolor!}
 **height** +
-__optional__|Current height of the network|integer (int32)|
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]: Starting from this line, all below content is coming from the `swagger.yml` file, so no changes should be done here, as it is auto generated content. If necessary, create a PR to update the contents of `swagger.yml` in the lisk-sdk repository.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Overall looking really good!
Just a few things I found, which I mentioned in the comments.

File: modules/ROOT/pages/references/api-specification.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/references/api-specification.adoc]: OK, it should be fine now..

File: modules/ROOT/pages/references/changelog.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/references/changelog.adoc]: OK, this should also be fine now.

File: modules/ROOT/pages/references/changelog.adoc [hunk 5]
```diff
@@ -116,7 +116,7 @@ It is utilized to vote for the forging genesis delegates in a convenient way dur
 == Database changes
 
 To see what has changed in the database layer of Lisk SDK 3.x, compare the 2 figures `Database layer SDK 2.x`
- and `Database layer SDK 3.x`
+and `Database layer SDK 3.x`
```
Comment (Tschakki) [inline modules/ROOT/pages/references/changelog.adoc]: Please use 1 line per sentence.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking great! Just one very minor issue left 👍

File: modules/ROOT/pages/references/changelog.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/references/changelog.adoc]: Good catch Mona! Now done.. 👍

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**